tests/bin/external_trogdor_command_example.py

```diff
@@ -20,19 +20,22 @@
 
 #
 # This is an example of an external script which can be run through Trogdor's
-# ExternalCommandWorker.
+# ExternalCommandWorker. It parses the custom workload JSON and sleeps for the given amount of time.
 #
 
 if __name__ == '__main__':
     # Read the ExternalCommandWorker start message.
     line = sys.stdin.readline()
     start_message = json.loads(line)
     workload = start_message["workload"]
-    print("Starting external_trogdor_command_example with task id %s, workload %s" \
-        % (start_message["id"], workload))
+    print("Starting external_trogdor_command_example with task id %s, workload %s"
+          % (start_message["id"], workload))
     sys.stdout.flush()
-   `print(json.dumps({"status": "running"}))`
+
+    # pretend to start some workload
+    print(json.dumps({"status": "running"}))
```

Comment:
the backticks in python 2 are equivalent to calling the `repr()` [function ](https://www.tutorialspoint.com/What-does-the-repr-function-do-in-Python-Object-Oriented-Programming) which returns a string out of the result of the `print()` call. I'm nt sure why this was added initially as it has no effect whatsoever

Reply:
Heh, I'm not sure how that got in there.  Thanks for removing it.